operator-common/src/test/java/io/strimzi/operator/common/operator/MockCertManager.java

```diff
@@ -315,7 +315,7 @@ public void generateSelfSignedCert(File keyFile, File certFile, Subject sbj, int
      * @param certFile path to the file which will contain the new self signed certificate
      * @param sbj      subject information
      * @param days     certificate duration
-     * @throws IOException
+     * @throws IOException  during writing to file
```

Comment:
You should start with a capital letter. Maybe you can also add a verb?

```suggestion
     * @throws IOException  Thrown when writing to file fails
```